spacecmd/spacecmd.spec

```diff
@@ -42,13 +47,19 @@ Name:           spacecmd
 Version:        4.0.12
 Release:        1%{?dist}
 Summary:        Command-line interface to Spacewalk and Red Hat Satellite servers
+%if %{_vendor} == "debbuild"
+Packager:        Uyuni packagers <uyuni-devel@lists.opensuse.org>
+Group:           kernel
+License:         GPL-2.0
```

Comment:
This cannot be, the license in RPM is "GPL-3.0-or-later"

Reply:
Right, and he Group is not correct neither.

Fixing it right now :-)